### ISSUE TITLE
Load translations from files

### ISSUE THREAD

**Issue by pavog** [necordjs/localization#187]
Is there an easy way to load translations from files (e.g. .json files)?
If yes, please tell me how to use it and/or add it to the documentation.
If no, could you please add such a feature? Thanks!

**Comment by SocketSomeone** [comment]
Yes, you can, use `useFactory` way when you registering module as `forRootAsync` and pass yours parsed json file to locales option. 

Also you can override this getLocales function from [this](https://github.com/necordjs/localization/blob/master/test/application.spec.ts) code
